tests/ops/api/v1/endpoints/test_privacy_request_endpoints.py

```diff
@@ -1322,6 +1334,41 @@ def test_get_failed_request_resume_info_from_email_send(
         }
         assert data["resume_endpoint"] == f"/privacy-request/{privacy_request.id}/retry"
 
+    @pytest.mark.parametrize(
```

Comment:
Making use of the parametrization, I like it!

Reply:
Thanks! I've been looking for opportunities to use it